packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/representations/services/DiagramFilterRefreshPolicyProvider.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.views.representations.services;
+
+import org.eclipse.sirius.components.collaborative.api.ChangeKind;
+import org.eclipse.sirius.components.collaborative.api.IRepresentationRefreshPolicy;
+import org.eclipse.sirius.components.collaborative.api.IRepresentationRefreshPolicyProvider;
+import org.eclipse.sirius.components.collaborative.diagrams.DiagramChangeKind;
+import org.eclipse.sirius.components.representations.IRepresentationDescription;
+import org.springframework.stereotype.Service;
+
+/**
+ * The representation refresh policy provider for the "Diagram Filter" form representation.
+ *
+ * @author gdaniel
+ */
+@Service
+public class DiagramFilterRefreshPolicyProvider implements IRepresentationRefreshPolicyProvider {
+
+
+    @Override
+    public boolean canHandle(IRepresentationDescription representationDescription) {
+        return "diagram_filter_form_description".equals(representationDescription.getId());
```

Comment:
Shouldn't it be `DiagramFilterDescriptionProvider.FORM_DESCRIPTION_ID` instead of `"FORM_DESCRIPTION_ID"`?

Reply:
done